### ISSUE TITLE
test: waitThrottleDelayBetweenWorkUnits is flaky

### ISSUE THREAD

**Issue by aeschli** [microsoft/vscode#230366]
https://dev.azure.com/monacotools/Monaco/_build/results?buildId=296767&view=logs&j=78322c78-e078-5e20-184d-1b2bb6aed118&t=6d18ead1-26fd-5e07-64e3-cefc56c57b3e

Also see https://github.com/microsoft/vscode/pull/230335

**Comment by aeschli** [comment]
I commented out the test for now.

**Comment by lramos15** [comment]
@bpasero Do you have any suggestions? This is a bit difficult because of the timeout and having to ensure that enough time has passed and the runOnceScheduler has executed which can happen pretty much anytime after the throttle delay

**Comment by bpasero** [comment]
Does it help to use `runWithFakedTimers`?

**Comment by aeschli** [comment]
Still flaky:
https://dev.azure.com/monacotools/Monaco/_build/results?buildId=297083&view=results
https://dev.azure.com/monacotools/Monaco/_build/results?buildId=297169&view=results

**Comment by aeschli** [comment]
Commented out again: [#230464](https://github.com/microsoft/vscode/pull/230464)

**Comment by lramos15** [comment]
Ugh, sorry about that. Thanks. Will investigate again, I wasn't able to get `runWithFakedTimers` to work, so instead I was trying to rely on promises and ensuring a certain amount of milliseconds has passed before the worker executed, but I guess that's still flaky as well

**Comment by bpasero** [comment]
Maybe we skip this task for now and do not invest too much in it, esp. since the option is not used in VS Code.

**Comment by lramos15** [comment]
Yeah that's fine, we can leave it skipped for now. My worry would be that if someone consumes it then it should be well tested. I'm confident the feature is working as intended currently, but the flakiness of the test is definitely frustrating. Maybe I can write a test for it within the copilot repo since that's where it is being consumed anyways.
